kani-driver/src/cbmc_property_renderer.rs

```diff
@@ -240,6 +240,12 @@ pub fn format_result(properties: &Vec<Property>, show_checks: bool) -> String {
     let mut number_tests_undetermined = 0;
```

Comment:
I wonder if we should rename this to `number_checks_*` to contrast with `number_covers_*`.

Reply:
Done.